compare_style.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
```

Comment:
This shell script seems to be more of a crutch for Travis since it doesn't support tracking build data.

I think codeclimate will be the better public service tracking linting metrics.

Plus we won't need to maintain another shellscript

Reply:
I appreciate being able to check my changes before submitting to an external service but it is optional for one to use. I would rather we have shared tooling than folks implementing their own ad-hoc methods. Indeed CodeClimate seems like a better service than Travis for tracking metrics on code quality though.